src/components/ButtonComponents/OperatorButtons/OperatorButton.js

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 
-const OperatorButton = () => {
-  return (
-    <>
-      {/* Display a button element rendering the data being passed down from the parent container on props */}
-    </>
-  );
+const OperatorButton = ({ operator }) => {
```

Comment:
Nice destructuring. Do you prefer this or passing props for a small assignment like this?